source/installation-guide/installing-wazuh-manager/linux/amazon/wazuh_server_packages_amazon.rst

```diff
@@ -88,11 +71,11 @@ Installing the Wazuh API
       # service wazuh-api status
```

Comment:
With the new packages changes, the status must be checked with wazuh-manager status.

Reply:
To be changed in corresponding issue https://github.com/wazuh/wazuh/issues/5824